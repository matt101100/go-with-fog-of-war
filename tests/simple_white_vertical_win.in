place A1
place B1
place J5
place B2
place A2
place B3
place A3
place B4
place A4
place B5
