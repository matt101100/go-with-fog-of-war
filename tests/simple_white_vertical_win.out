White wins!
A1B1J5B2A2B3A3B4A4B5
Thank you for playing!
